src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 400;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
```

this comment was to make like this
```suggestion
html {
    font-family: "Avenir", sans-serif;
   font-weight: 300;
}

* {
  box-sizing: border-box;

```